### ISSUE TITLE
Question on Using Text Pattern to Filter Data

### ISSUE THREAD

**Issue by Masahiroshi** [VolkovLabs/business-satellite#63]
I've followed your tutorial on using the text pattern function to filter on specific values as well as using grafana variables.
I was wondering if the text pattern had the ability to filter across multiple parameters or is there a specific syntax to pass it multiple parameters?

For example,
I got an OR function to work by entering the following examples:
 metric_type = discrete | metric_aggregate = mean

I get the equivalent of metric_type equals 'discrete' OR metric_aggregate equals 'mean'

And if I used a set of variables:
metric_type = ${metric_type:regex} | metric_aggregate = ${metric_aggregate:regex}

I get the equivalent of metric_type equals any values in variable $metric_type OR metric_aggregate equals any values in variable $metric_aggregate

But now I am looking for the ability to pass an AND function through:

Basically the same scenario stated above but with AND as the operator instead of OR.
The variable scenario is what I'm trying to accomplish:

metric_type = ${metric_type:regex} & metric_aggregate = ${metric_aggregate:regex}

I want the equivalent of metric_type equals any values in variable $metric_type and metric_aggregate equals any values in variable $metric_aggregate

I am not very fluent in regex so there may be a syntax that I am unfamiliar with that will satisfy my case.
But, I was wondering if the Text Pattern function had the ability to resolve multiple parameters by separating the statement with a specific character or if you could point me in the direction of a specific regex syntax that would represent the equivalent of an AND operator.

**Comment by mikhail-vl** [comment]
We are announcing the retirement of the Business Satellite data source, effective as of today, July 12, 2025, as part of our commitment to maintaining a modern and efficient Grafana plugin ecosystem. To ensure continued access to your data and analytics, we strongly recommend migrating to the Infinity data source, a powerful and flexible alternative designed to meet your evolving needs.

### Why This Change?

The retirement of the Business Satellite data source reflects our focus on consolidating resources toward scalable, future-proof solutions. The Infinity data source provides enhanced support for diverse data formats, real-time updates, and seamless integration with Grafana 11 and 12, addressing limitations in the older plugin.
